src/Notifications.svelte

```diff
@@ -3,7 +3,12 @@
 		<li class="toast" style="background: {toast.background};" out:animateOut>
 			<div class="content">
 				{toast.msg}
-			</div>
+      </div>
+      {#if toast.closeable}
```

Comment:
Looks like a good addition - I think closable toasts should probably be allowed without  a timeout too, so that they **have** to be closed. Not sure how we'd express that in the API.